core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -317,10 +317,22 @@ var TopicsAndSkillsDashboardPage = function() {
     var handles = await browser.getAllWindowHandles();
     initialHandles = handles;
     var parentHandle = await browser.getWindowHandle();
-    await waitFor.elementToBeClickable(
-      createSkillButton,
-      'Create Skill button takes too long to be clickable');
-    await createSkillButton.click();
+    var introCardCreateSkillButtonPresent = await createSkillButton.isPresent();
```

Comment:
Can we make sure that the page has loaded before this check? Otherwise we might decide the button isn't present just because it hasn't loaded.